### PR TITLE
Cleric Changes and Rebalance (Stats, Armor Training, Weapon Skills)

### PR DESCRIPTION
**Pull request by Optimism333**
## About The Pull Request
Clerics have needed a good and hard rebalancing for a LONG while. I finally got around to just doing it myself.

Unlike my changes to warrior, this could definitely classify as an overall nerf, but it was very much deserved, and not all of the changes were nerfs.

### Significant Changes ###
- Their mace skill has been reduced to skilled instead of expert, though it can still be boosted to expert if playing an old cleric.
- Their armor training has been changed to medium instead of heavy. None of the armor they start with is heavy, and heavy armor training should overall be rarer in my opinion.
- Their bonuses to Intelligence, Constitution, and Endurace have each been reduced by 1. Instead of 2, 2, and 3 it is now 1, 1, and 2.
- Their -2 to Perception has been removed.
- Like templar, they get a psicross AND surcoat based on their chosen diety (in most cases).
- They now also start with a gorget.
- They start with a satchel and a poor pouch of coins.
- Gave their description a little retouch: "Clerics are wandering warriors of the Gods, drawn from the ranks of temple acolytes who demonstrated martial talent. Protected by armor and zeal, they are a force to be reckoned with."
- With clerics getting nerfed in this pr, and their devotion being potentially capped in another, I bumped their available slots from 2 to 4

Screenshot (247)

## Why It's Good For The Game
- Gives a look at rebalancing what was likely the most unbalanced common combat drifter role, and brings it up to speed with some of the new religious customization.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/cleric.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //shield
 /datum/advclass/combat/cleric
 	name = "Cleric"
-	tutorial = "Clerics are wandering warriors of the Gods, an asset to any party."
+	tutorial = "Clerics are wandering warriors of the gods, drawn from the ranks of temple acolytes who demonstrated martial talent. Protected by armor and zeal, they are a force to be reckoned with."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/cleric.dm:4]: Capitalise Gods?

Comment (Optimism333) [reply code/modules/jobs/job_types/roguetown/adventurer/types/combat/cleric.dm]: Done.

**Comment by Optimism333** [comment]
Decided to not give cleric chain leggings, and gave them their old leather pants back. The steel half-plate they start with and a gorget is plenty generous, already. Unlike warrior, they are limited to two slots so it's not outrageous, but they're still a common drifter class and if they want better leg protection they can go to a smith.

**Comment by YuiY1997** [comment]
> Decided to not give cleric chain leggings, and gave them their old leather pants back. The steel half-plate they start with and the added gorget is plenty generous, already. Unlike warrior, they are limited to two slots so it's not outrageous, but they're still a common drifter class and if they want better leg protection they can go to a smith.

isn't steel halfplate heavy armor?

**Comment by YuiY1997** [comment]
Cleric was limited to two slots because they were OP. After nerfing them, I think it's worth considering raising them to more than two slots, especially since clerics are not necessarily affiliated with the church itself as an adventurer class.

**Comment by Optimism333** [comment]
Four slots, now

**Review by Stutternov — APPROVED** [review]

**Pull request merged**